classes/components/forms/publication/CatalogEntryForm.inc.php

```diff
@@ -51,7 +51,7 @@ public function __construct($action, $locales, $publication, $submission, $baseU
 			$series = $result->next();
 			$seriesOptions[] = [
 				'value' => (int) $series->getId(),
-				'label' => $series->getLocalizedTitle(),
+				'label' => $series->getLocalizedTitle().(($series->getIsInactive())?' (Inactive)':''),
```

Comment:
I should have caught this on the OJS code review, but this needs to use a locale key so that it can be translated. Put the section name and the inactive fragment into the same locale string, "{$section} (Inactive)", so that it can be translated properly even if another language wants to reorder it.